code/game/objects/items/devices/radio/headset.dm

```diff
@@ -183,6 +183,13 @@
 	item_state = "headset"
 	ks2type = /obj/item/encryptionkey/headset_med
 
+/obj/item/radio/headset/headset_med/para
+	name = "paramedic radio headset"
+	desc = "A headset for the trusty paramedic, NanoTrasen search and rescue."
```

Comment:
```suggestion
	desc = "A headset for the trusty paramedic, Nanotrasen search and rescue."
```
As per Lore Team, this should be "Nanotrasen".
(Which means I have a new PR to make, guh.)

Reply:
On it.